astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,493 @@
+            )
+
+        # configure request_payload for ephemeris query
+        # thankfully, adding extra planet-specific keywords here does not break query for other planets
+        request_payload = OrderedDict(
+            [
+                ("abbrev", planet.lower()[:3]),
+                ("ephem", conf.planet_defaults[planet]["ephem"],),
+                ("time", obs_time),  # UTC. this should be enforced when checking inputs
+                (
+                    "fov",
+                    10,
+                ),  # next few are figure options, can be hardcoded and ignored
+                ("fov_unit", planet.capitalize() + " radii"),
+                ("center", "body"),
+                ("center_body", planet.capitalize()),
+                ("center_ansa", conf.planet_defaults[planet]["center_ansa"]),
+                ("center_ew", "east"),
+                ("center_ra", ""),
+                ("center_ra_type", "hours"),
+                ("center_dec", ""),
+                ("center_star", ""),
+                ("viewpoint", viewpoint),
+                (
+                    "observatory",
+                    "Earth's center",
+                ),  # has no effect if viewpoint != observatory so can hardcode. no plans to implement calling observatories by name since ring node only names like 8 observatories
+                ("latitude", latitude),
+                ("longitude", longitude),
+                ("lon_dir", "east"),
+                ("altitude", altitude),
+                ("moons", conf.planet_defaults[planet]["moons"]),
+                ("rings", conf.planet_defaults[planet]["rings"]),
+                ("arcmodel", conf.neptune_arcmodels[int(neptune_arcmodel)]),
+                (
+                    "extra_ra",
+                    "",
+                ),  # figure options below this line, can all be hardcoded and ignored
+                ("extra_ra_type", "hours"),
+                ("extra_dec", ""),
+                ("extra_name", ""),
+                ("title", ""),
+                ("labels", "Small (6 points)"),
+                ("moonpts", "0"),
+                ("blank", "No"),
+                ("opacity", "Transparent"),
+                ("peris", "None"),
+                ("peripts", "4"),
+                ("arcpts", "4"),
+                ("meridians", "Yes"),
+                ("output", "html"),
+            ]
+        )
+
+        # return request_payload if desired
+        if get_query_payload:
+            return request_payload
+
+        # set return_raw flag, if raw response desired
+        if get_raw_response:
+            self.return_raw = True
+
+        # query and parse
+        response = self._request(
+            "GET", URL, params=request_payload, timeout=self.TIMEOUT, cache=cache
+        )
+        self.uri = response.url
+
+        return response
+
+    def _parse_ringnode(self, src):
```

Comment:
I think it's cleaner if the full response is passed to this method rather than `response.text`, it will be slightly easier to debug